lib/version.rb

```diff
@@ -1,3 +1,3 @@
-module BugsnagMazeRunner
+module Maze
```

Comment:
This just bring it in line with an evolving convention elsewhere in the codebase.